### DISCUSSION TITLE
twinBASIC BETA 77

### DISCUSSION THREAD

**Discussion by WaynePhillipsEA** [twinbasic/twinbasic#994] [Releases]
- fixed: codegen issues around fixed-length strings [ baka, discord ]

<hr /><em>This discussion was created from the release <a href='https://github.com/twinbasic/twinbasic/releases/tag/'>twinBASIC BETA 77</a>.</em>

**Comment by MatTheCatz** [comment]
Hi, can we open *.ctl from vb6 with twinBASIC?

**Reply by WaynePhillipsEA** [reply]
Not at the moment.  You can import a VBP, but it currently will only import the BAS/CLS files, not forms and user controls etc.   Support for this is high on the priority list.

**Reply by MatTheCatz** [reply]
Do you know when a realise have this issues?

**Reply by WaynePhillipsEA** [reply]
Check back in a month

**Comment by MatTheCatz** [comment]
Perfect

**Comment by MatTheCatz** [comment]
when i open a .dll project from VB6, 
I have a unrecognized symbol with this line 
'App.LogEvent vLogEventModule + vLogEventSQL + vLogEvent, vbLogEventTypeError'
vbLogEventTypeError is not recongnize

**Reply by WaynePhillipsEA** [reply]
`App` is only partially implemented at the moment, sorry.  See #143

**Comment by MatTheCatz** [comment]
Some time err.description become DATAription
